modules/webgl/src/classic/typed-array-utils.ts

```diff
@@ -9,6 +9,7 @@ const ERR_TYPE_DEDUCTION = 'Failed to deduce GL constant from typed array';
 /**
  * Converts TYPED ARRAYS to corresponding GL constant
  * Used to auto deduce gl parameter types
+ * @deprecated Use getDataTypeFromTypedArray
```

Comment:
I guess one would need two functions: `getDataTypeFromTypedArray` and `getGLFromVertexType`

Note to self: "Vertex Type" being "Normalized Data Type" which is indeed confusing - might want to consider renaming VertexType to `NormalizedDataType`